test/cmdlineTests/function_debug_info/exit

```diff
@@ -0,0 +1 @@
+0
```

Comment:
This file is optional when return code is 0. I'd remove it to decrease the number of files here.

Same in the other test.